libs/compress/src/main/java/org/opensearch/package-info.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/**
+ * This is the compress library for registering optional
+ * {@link org.opensearch.core.compress.Compressor} implementations
+ */
+package org.opensearch;
```

Comment:
Should this be in the `org.opensearch.compress` folder instead?

Reply:
Missing javadoc checker requires a `package-infol.java` in each package.

Reply:
Even for a package that contains no code? A quick test suggests this isn't needed. Also the comment isn't technically accurate as applied to the `org.opensearch` package.